bpfman/Cargo.toml

```diff
@@ -53,6 +53,7 @@ oci-distribution = { workspace = true, default-features = false, features = [
     "native-tls",
     "trust-dns",
 ] }
+openssl = { workspace = true, features = ["vendored"] }
```

Comment:
Hrm I don't think we can rely on the vendored version of openssl for the RPM. see

https://github.com/bpfman/bpfman/pull/1023

@danielmellado Maybe you have an idea? 

I'm not quite sure why this is breaking us only on ARM

Reply:
Removed.